### PR TITLE
Python package "lockfile" needs package "pbr"

### PR DESCRIPTION
**Pull request by hajgato**
Lockfile installs pbr from pip, so we do not have any control on the version of pbr (as it installed always the newest version) Plus python installation fails on nodes without (or with restricted) internet access) 
I do not know what versions of Lockfile affected, I have only modified the latest Python versions.

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-2.7.9-goolf-1.5.14.eb [hunk 1]
```diff
@@ -55,6 +55,9 @@ exts_list = [
     ('argparse', '1.2.1', {
         'source_urls': ['http://argparse.googlecode.com/files/'],
     }),
+    ('pbr', '0.10.7', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.9-goolf-1.5.14.eb]: there's a `0.10.8` available?

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by hajgato** [comment]
@boegel pbr version updated, plus  two more modules inserted needed by matplotlib (mock, pytz)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8660d0f962ba0e7332ee for a full test report.

**Comment by boegel** [comment]
Looks good, thanks @hajgato!

**Pull request merged**